onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc

```diff
@@ -720,7 +720,9 @@
         "^test_constantofshape_int_zeros",
         "^test_reduce_log_sum_empty_set_cpu",
         "^test_reduce_log_sum_exp_empty_set_cpu",
-        "^test_reduce_prod_empty_set_cpu"
+        "^test_reduce_prod_empty_set_cpu",
+        //DML EP does not execute operators with an empty input tensor
```

Comment:
Should we add a bug note here to indicate that this should be resolved as a graph implementation that returns an constant inf tensor strided appropriately?